Two sgen msbuild properties on the page are for desktop use only (#3010)
MicrosoftDocs/visualstudio-docs: docs/msbuild/common-msbuild-project-properties.md

| ProduceReferenceAssembly | A boolean value that when set to `true` enables production of [reference assemblies](https://github.com/dotnet/roslyn/blob/master/docs/features/refout.md) for the current assembly. `Deterministic` should be `true` when using this feature. This property corresponds to the `/refout` switch of the *vbc.exe* and *csc.exe* compilers. |
| ProduceOnlyReferenceAssembly | A boolean value that instructs the compiler to emit only a reference assembly rather than compiled code. Cannot be used in conjunction with `ProduceReferenceAssembly`.  This property corresponds to the `/refonly` switch of the *vbc.exe* and *csc.exe* compilers. |
| RemoveIntegerChecks | A boolean value that indicates whether to disable integer overflow error checks. The default value is `false`. This property is equivalent to the `/removeintchecks` switch of the *vbc.exe* compiler. |

> | GenerateSerializationAssemblies | Indicates whether XML serialization assemblies should be generated by *SGen.exe*, could be set to on, auto or off. This property is used for assemblies that target .NET Framework only. To generate XML serialization assemblies for .NET Standard or .NET Core assemblies, reference the *Microsoft.XmlSerializer.Generator* NuGet package. |

Reviewer: ```suggestion
| GenerateSerializationAssemblies | Indicates whether XML serialization assemblies should be generated by *SGen.exe*, which can be set to on, auto, or off. This property is used for assemblies that target .NET Framework only. To generate XML serialization assemblies for .NET Standard or .NET Core assemblies, reference the *Microsoft.XmlSerializer.Generator* NuGet package. |
```

> | SGenUseProxyTypes | A boolean value that indicates whether proxy types should be generated by *SGen.exe*. Same as *GenerateSerializationAssemblies*, this property is used for .NET Framework only.<br /><br /> The SGen target uses this property to set the UseProxyTypes flag. This property defaults to true, and there is no UI to change this. To generate the serialization assembly for non-webservice types, add this property to the project file and set it to false before importing the *Microsoft.Common.Targets* or the *C#/VB.targets*. |

Reviewer 2: ```suggestion
| SGenUseProxyTypes | A boolean value that indicates whether proxy types should be generated by *SGen.exe*. This applies only when *GenerateSerializationAssemblies* is true, and for .NET Framework only.<br /><br /> The SGen target uses this property to set the UseProxyTypes flag. This property defaults to true, and there is no UI to change this. To generate the serialization assembly for non-webservice types, add this property to the project file and set it to false before importing the *Microsoft.Common.Targets* or the *C#/VB.targets*. |
```

?

> | SGenToolPath | An optional tool path that indicates where to obtain *SGen.exe* when the current version of *SGen.exe* is overridden. Same as *GenerateSerializationAssemblies*, this property is used for .NET Framework only.|

Reviewer: ```suggestion
| SGenToolPath | An optional tool path that indicates where to obtain *SGen.exe* when the current version of *SGen.exe* is overridden. This property is used for .NET Framework only.|
```

Reviewer: I believe this is what is meant here by "Same as *GenerateSerializationAssemblies*?
